native/src/components/Categories.tsx

```diff
@@ -81,7 +81,7 @@ const Categories = ({
       path={category.path}
       AfterContent={category.organization && <OrganizationContentInfo organization={category.organization} />}
       Footer={children.map(it => (
```

Comment:
So in theory we have a list of lists here, there is multiple categories that are mapped to `SubCategoryList`s. In theory the childrens should also be rendered in a list. Can also be done in a separate issue though.

Reply:
If you don't want to do it in this PR we can just open a new issue for this :)

Reply:
@steffenkleinle I implemented it :)